<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Console</class>
 <widget class="QWidget" name="Console">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Console virtuelle</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QSlider" name="vsCh1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="vsCh2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QSlider" name="vsCh5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSlider" name="vsCh3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QSlider" name="vsCh4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSpinBox" name="sbCh1">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="sbCh2">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="spCh3">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSpinBox" name="sbCh4">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QSpinBox" name="sbCh5">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QSpinBox" name="sbOffset"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>vsCh1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sbCh1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>43</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbCh1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>vsCh1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vsCh2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sbCh2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbCh2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>vsCh2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vsCh3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spCh3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spCh3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>vsCh3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbCh4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>vsCh4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vsCh4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sbCh4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbCh5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>vsCh5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vsCh5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sbCh5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
